udf_js/json_extract_events.sql

```diff
@@ -0,0 +1,85 @@
+CREATE TEMP FUNCTION
+  udf_js_json_extract_events (input STRING)
+  RETURNS ARRAY<STRUCT<
+  event_process STRING,
+  event_timestamp INT64,
```

Comment:
Is this timestamp measured based on the ping start time? If it's an epoch time, it would be nice to output a TIMESTAMP type.

Reply:
It's milliseconds since session start time, unfortunately. A view that includes a column where those two are added and cast to timestamp would be a very nice enhancement though.